From cfbe30990f2b2775638caae1208e686d5d30c8b3 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 28 Nov 2018 23:55:47 +0200
Subject: [PATCH] version: date: fix the date format

Current date format: dd MMM YYYY.
It is the only FW component which has the days first in the date.
All other components have the month first in the date.
New date formate: MMM dd YYYY.

Change-Id: I476786e0dbe3e0a3488e8047d8ad3d25f001b8e2
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/Makefile b/Makefile
index 23a6920..291e074 100644
--- a/Makefile
+++ b/Makefile
@@ -40,7 +40,7 @@
 
 $(AUTO_GENERATED_FILE): localversion-habanalabs $(PREBOOT_SRCS_C) $(PREBOOT_SRCS_S)
 	@( printf '#define PREBOOT_VER "%s"\n' '$(shell cut -d- -f2- localversion-habanalabs)' ) > $@
-	@date +'#define PREBOOT_DATE "%d %b %C%y"' >> $@
+	@date +'#define PREBOOT_DATE "%b %d %C%y"' >> $@
 	@date +'#define PREBOOT_TIME "%T"' >> $@
 
 %.o: %.c $(AUTO_GENERATED_FILE)
